packages/design/src/Elevations.mdx

```diff
@@ -26,45 +22,88 @@ We also include a _slight_ y-axis offset of elevation shadows to imply that the
 light source is coming from an angle slightly over the head of a user looking
 directly into the screen.
 
-## Web
+### Low
+
+<Example of="low" />
+
+A low element may have a slight elevation, but is not necessarily something that
+will glide across the surface. A card that holds interactive controls within
+itself, but that does not have any interactions itself, may be useful to
+indicate as "low".
```

Comment:
Can you give an example as you did for base? Its a lil hard for me to know from the description

Reply:
Also I know you gave an example here in a way, but i don't understand what "does not have any interaction itself" means

Reply:
I left a comment on your DSC - Mobile Card file, but... basically a mobile card which doesn't technically exist yet 😰

Reply:
ahhh i see i see. okay i think i get it!